ipn/ipnlocal/web_stub.go

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+//go:build ios
```

Comment:
android?

Reply:
oh and doesn't tvOS use `ios`?

Reply:
> oh and doesn't tvOS use ios?

Yeah I think so. That's alright for now.